Dockerfile

```diff
@@ -7,12 +7,23 @@ ENV PYTHONUNBUFFERED 1
 
 # Install non python dependencies
 RUN apt-get update
-RUN apt-get install -y libpq-dev gcc curl
+RUN apt-get install -y libpq-dev gcc curl gpg
 
 # Fetch Amazon RDS certificate chain
 RUN curl https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem -o /usr/local/share/amazon-certs.pem
 RUN chmod a=r /usr/local/share/amazon-certs.pem
 
+# Install MSSQL ODBC 18 Driver
+RUN mkdir -m 0755 -p /etc/apt/keyrings/
+RUN curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg
+RUN echo "deb [signed-by=/etc/apt/keyrings/microsoft.gpg] https://packages.microsoft.com/debian/12/prod bookworm main" | tee /etc/apt/sources.list.d/mssql-release.list
```

Comment:
Open to suggestions on how to add this microsoft gpg key, but again this seems the most straightforward / secure.

Reply:
i think this is fine. maybe a few comments about what the curl and echo statements are doing, but this seems fine to me.

Reply:
it might be a good idea while we're here to add comments about why each dependency is needed. like this one is only going to be used for the TransitMaster ingestion application. Future LAMP will be happy that they know why each dependency was added to this shared dockerfile.

Reply:
one last note is that it seems like we should chain all these commands together as [per dockers best practices](https://stackoverflow.com/a/39234643)

```
# Install necessary tools and MSSQL ODBC 18 Driver
RUN mkdir -m 0755 -p /etc/apt/keyrings/ \
    && curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg \
    && echo "deb [signed-by=/etc/apt/keyrings/microsoft.gpg] https://packages.microsoft.com/debian/12/prod bookworm main" | tee /etc/apt/sources.list.d/mssql-release.list \
    && apt-get update \
    && ACCEPT_EULA=Y apt-get install -y msodbcsql18
```

Reply:
Incorporated.